clients/web/src/views/body/UsersView.js

```diff
@@ -42,6 +43,16 @@ var UsersView = View.extend({
             parentView: this
         });
 
+        this.sortCollectionWidget = new SortCollectionWidget({
+            collection: this.collection,
+            parentView: this,
+            fields: {
+                lastName: 'Last Name',
+                created: 'Creation Date',
+                size: 'Used Space'
```

Comment:
`size` and `created` are not indexed fields at the moment, we should add indexes to them in the user model.